src/haddock/modules/analysis/clustfcc/__init__.py

```diff
@@ -163,11 +163,10 @@ def _run(self):
             for cluster_rank, _e in enumerate(sorted_score_dic, start=1):
                 cluster_id, _ = _e
                 # sort the models by score
-                model_score_l = [(e.score, e) for e in clt_dic[cluster_id]]
-                model_score_l.sort()
+                clt_dic[cluster_id].sort()
                 # rank the models
-                for model_ranking, element in enumerate(model_score_l, start=1):
-                    score, pdb = element
+                for model_ranking, pdb in enumerate(clt_dic[cluster_id],
```

Comment:
:coffee: :snake: no need to change anything

I really dislike it if we are so strict with the 80 chars, even though if I am a 80 (almost 72) char maniac. In this case, I suggest one of the following:

```python
for model_ranking, pdb in enumerate(clt_dic[cluster_id], start=1):  # noqa :E501
```


```python
cluster_id_ = clt_dic[cluster_id]
for model_ranking, pdb in enumerate(cluster_id_, start=1):
```


```python
_gen = enumerate(clt_dic[cluster_id], start=1)
for model_ranking, pdb in _gen:
```

I think with the above the code becomes more readable if you have to increase the number of parameters of the generator.